qa/integration/src/test/resources/features/authorization/GroupService.feature

```diff
@@ -256,3 +256,792 @@ Feature: Group Service tests
+  Kapua should throw Exception.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    Given I expect the exception "KapuaIllegalArgumentException" with the text "*"
+    When I create the group with name "Group@123" and description "Valid description 123"
+    Then An exception was thrown
+    Then I logout
+
+  Scenario: Creating Group Without a Name And With Valid Description
+  Login as kapua-sys, go to groups, try to create a group without a name and valid description.
+  Kapua should throw Exception.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    Given I expect the exception "KapuaIllegalNullArgumentException" with the text "*"
+    When I create the group with name "" and description "Valid description 123"
+    Then An exception was thrown
+    Then I logout
+
+  Scenario: Creating Group With Numbers In Name With Valid Description
+  Login as kapua-sys, go to groups, try to create a group with name that contains numbers and valid description.
+  Kapua should not return any errors.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    When I create the group with name "Group1234567890" and description "Valid description 123"
+    Then No exception was thrown
+    When I create the group with name "1234567890" and description "Valid description 123"
+    Then No exception was thrown
+    Then I logout
+
+  Scenario: Creating Unique Group With Short Description
+  Login as kapua-sys, go to groups, try to create a group with short description.
+  Kapua should not return any errors.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    When I create the group with name "Group1" and description "a"
+    Then No exception was thrown
+    Then I logout
+
+  Scenario: Creating Unique Tag With Long Description
```

Comment:
Please change this title to: "Creating Unique Group With Long Description" becasue we are creating groups here and not tags =)